js/index.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Plantalytics
+ *     Copyright (c) 2016 Sapphire Becker, Katy Brimm, Scott Ewing,
+ *       Matt Fraser, Kelly Ledford, Michael Limb, Steven Ngo, Eric Turley.
+ *     This project is licensed under the MIT License.
+ *     Please see the file LICENSE in this distribution for license terms.
+ * Contact: plantalytics.capstone@gmail.com
+ */
+
+$(function() {
+    /*
+     * Listener for 'enter' keypress on #loginUsername field.
+     */
+    $('#loginUsername').on('keypress', function(evt) {
+        if (evt.which == 13 && !evt.shiftKey) {
+            $('#loginButton').trigger('click');
+        }
+    });
+
+    /*
+     * Listener for 'enter' keypress on #loginPassword field.
+     */
+    $('#loginPassword').on('keypress', function(evt) {
+        if (evt.which == 13 && !evt.shiftKey) {
+            $('#loginButton').trigger('click');
+        }
+    });
+
+    $('#loginButton').click(function() {
+        /* This function to handle user login
+         * once backend support allows
+         */
+        $.ajax({
+            url: backendIpAddress + 'login?username=' + $('#loginUsername').val()
```

Comment:
GET params should be removed and changed to a data object, can see the uses in dashboard-controls.js for an example.